src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml

```diff
@@ -25,40 +25,76 @@
                     </p>
                 </div>
             </details>
-            <p>You can browse all opportunities available.</p>
-            <p>Some businesses have chosen not to share their organisation's name publicly so these opportunities just show a reference number. If your application is successful, we'll let you know who the employer willing to fund your apprenticeship training is.</p>
-            <h2 class="govuk-heading-m">@Model.Opportunities.Count() @(Model.Opportunities.Count() == 1 ? "opportunity" : "opportunities")</h2>
-            <div class="das-search-results">
-                <ol class="das-search-results__list">
-                    @foreach (var opportunity in Model.Opportunities)
-                    {
-                    <li class="das-search-results__list-item">
-                        <h2 class="govuk-heading-m das-search-results__heading govuk-!-margin-bottom-0">
-                            <a href="@Url.Action("Detail", new { EncodedPledgeId = opportunity.ReferenceNumber })" class="das-search-results__link">@opportunity.EmployerName (@opportunity.ReferenceNumber)</a>
-                        </h2>
-                        <p class="govuk-hint">up to @opportunity.DisplayAmount available in @Model.TaxYear</p>
-                        <ul class="govuk-list govuk-!-margin-bottom-0">
-                            <li>
-                                <strong>Locations: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.Locations</span>
-                            </li>
-                            <li>
-                                <strong>Sectors: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.Sectors</span>
-                            </li>
-                            <li>
-                                <strong>Type of role: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.JobRoles</span>
-                            </li>
-                            <li>
-                                <strong>Levels: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.Levels</span>
-                            </li>
-                        </ul>
-                    </li>
-                    }
-                </ol>
-            </div>
         </div>
     </div>
+    <div class="govuk-grid-row">
+        <form method="post">
```

Comment:
My previous tech review requested that we not perform an HTTP Post in order to do this filtering.

Reply:
Apologies. This has been corrected now.